pkg/pods/pod.go

```diff
@@ -100,12 +101,12 @@ func (pod *Pod) Halt(manifest Manifest) (bool, error) {
 		switch err.(type) {
 		case nil:
 			// noop
-		case hoist.DisableError:
+		case launch.DisableError:
 			// do not set success to false on a disable error
-			pod.logLaunchableWarning(launchable.Id, err, "Could not disable launchable")
+			pod.logLaunchableWarning(launchable.GetID(), err, "Could not disable launchable")
 		default:
-			// this case intentionally includes hoist.StopError
-			pod.logLaunchableError(launchable.Id, err, "Could not halt launchable")
+			// this case intentionally includes launch.StopError
+			pod.logLaunchableError(launchable.GetID(), err, "Could not halt launchable")
```

Comment:
There's a naming conflict between `host.Launchable` having an `Id` field and an `ID()` method.  It's not a strict conflict in this case, but it's too close for my comfort.

But you reminded me that we do have a convention of interfaces with `ID()`, so I'm more inclined now to rename the field.